### PR TITLE
v8 macOS Update Create a web API

### LINKED ISSUE #29827: 8.0 Update: Create a web API: VS for Mac
The .NET 8 version of this tutorial needs to be updated for the Visual Studio for Mac tabs instruction, screen shots and code. Currently that version and tab just point to the .NET 7 version.

For previous version of .NET 8 previews, Visual Studio for Mac and Visual Studio for Mac 2022 preview was not compatible. Now it is as of VS for mac build 17.6.1 and .NET 8 preview 6

---
#### Document Details

⚠ *Do not edit this section. It is required for learn.microsoft.com ➟ GitHub issue linking.*

* ID: 583ae3e7-5915-f22b-e54c-6c100cf2a7da
* Version Independent ID: 95f2549e-29f9-e31a-b4d0-432b971643ff
* Content: [Tutorial: Create a web API with ASP.NET Core](https://learn.microsoft.com/en-us/aspnet/core/tutorials/first-web-api?view=aspnetcore-8.0&tabs=visual-studio-mac)
* Content Source: [aspnetcore/tutorials/first-web-api.md](https://github.com/dotnet/AspNetCore.Docs/blob/main/aspnetcore/tutorials/first-web-api.md)
* Product: **aspnet-core**
* Technology: **aspnetcore-tutorials**
* GitHub Login: @wadepickett
* Microsoft Alias: **wpickett**

---
[Associated WorkItem - 148469](https://dev.azure.com/msft-skilling/Content/_workitems/edit/148469)

### PR DESCRIPTION
**Pull request by wadepickett**
Fixes #29827 

Update Visual Studio for Mac instruction for .NET 8 and test both VS Code and VS for Mac where they overlap using the same tab.

---

#### Internal previews

| 📄 File | 🔗 Preview link |
|:--|:--|
| [aspnetcore/tutorials/first-web-api.md](https://github.com/dotnet/AspNetCore.Docs/blob/6a2b38e08edd83ad553ff9561f6e4fb3ecb00cfc/aspnetcore/tutorials/first-web-api.md) | [Tutorial: Create a web API with ASP.NET Core](https://review.learn.microsoft.com/en-us/aspnet/core/tutorials/first-web-api?branch=pr-en-us-30104) |

### CODE REVIEW

File: aspnetcore/tutorials/first-web-api.md [hunk 1]
```diff
@@ -98,19 +96,17 @@ A NuGet package must be added to support the database used in this tutorial.
...
+ * In Visual Studio for Mac 2022, select **File** > **New Project...**.
 
 * In the **Choose a template for your new project** dialog:
   * Select **Web and Console** > **App** > **API**.
   * Select **Continue**.
 
 * In the **Configure your new API** dialog, make the following selections:
-  * Confirm the **Target framework** is **.NET 7.0** (or later).
-  * Confirm the checkbox for **Use controllers (uncheck to use minimal APIs)** is checked.
+  * Confirm the **Target framework** is **.NET 8.0** (or later).
```
Comment (tdykstra) [inline aspnetcore/tutorials/first-web-api.md:106]: I'm not sure "or later" makes sense here. We don't say that in the prerequisites section, and we don't know what changes may come in .NET 9.

**Review by tdykstra — APPROVED** [review]
LGTM, one comment

File: aspnetcore/tutorials/first-web-api.md (hunk 1 above)
Comment (wadepickett) [inline aspnetcore/tutorials/first-web-api.md:106]:
```suggestion
  * Confirm the **Target framework** is **.NET 8.0**.
```

Comment (wadepickett) [reply aspnetcore/tutorials/first-web-api.md]: Corrected this. Thanks Tom!

**Pull request merged**